app/jobs/fs/submit_message_draft_job.rb

```diff
@@ -7,8 +7,9 @@ def perform(message_draft, bulk_submit: false, fs_client: FsEnvironment.fs_clien
     response = fs_api.post_submission(
       message_draft.form.identifier,
       Base64.strict_encode64(message_draft.form_object.content),
-      message_draft.form_object.is_signed,
-      message_draft.form_object.mimetype
+      allow_warn_status: false,
```

Comment:
Toto znamena, ze nepovolime odoslat podania, pri ktorych budu warningy? Alebo to ma nejaky iny vyznam?

Reply:
áno. Môžeme dať aj iný default ak chceš. Neviem, či to časom chceme aj nejako nastavovať z GUI per správa alebo čo

Reply:
Warn status by som povolila, podla toho, co sme sa s nimi bavili to je bezny pripad, ale ignoruju to a odosielaju aj tak. Aj teraz skusali take podat.

Reply:
ok, zmenil som